arcade/scene.py

```diff
@@ -20,17 +21,13 @@ class Scene:
     https://api.arcade.academy/en/latest/tutorials/views/index.html
 
     Attributes:
-        :sprite_lists: A list of `SpriteList` objects. The order of this list is the
-                       order in which they will be drawn.
-        :name_mapping: A dictionary of `SpriteList` objects. This contains the same lists
-                       as the `sprite_lists` attribute, but is a mapping of them by name. This is
-                       not necessarily in the same order as the `sprite_lists` attribute.
+        :DON'T CHANGE _sprite_lists or _name_mapping
```

Comment:
Just remove this whole attribute section. It's already implied that user should not access these members unless they know exactly what they are doing.

Reply:
Fixed